ui/src/components/UploadStory.tsx

```diff
@@ -0,0 +1,855 @@
+import {
```

Comment:
ok plz split this component, and keep it with the `Edit` page instead of storing it under `src/components/`

```
pages
  ...
  Edit -> folder
    index.tsx -> has the page component
    UploadStory.tsx -> upload story component
    ListboxComponent.tsx -> The react window logic
    types.ts -> has the types
    ... any other files you find needed
  ...
 ```

Reply:
Just to clarify, index.tsx is the page for edit story ONLY right (basically just move the current page in there)?

Reply:
yup